webpack/scenes/RedHatRepositories/RedHatRepositoriesPage.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint import/no-unresolved: [2, { ignore: [foremanReact/*] }] */
+/* eslint-disable import/no-unresolved */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'react-bootstrap';
```

Comment:
I think `patternfly-react` is using `react-bootstrap` already :)

Reply:
@laviro Yeah I remember looking at that too and thinking why bootstrap?  But see the previous comment :smiley: